introduction.md

```diff
@@ -1,2 +1,9 @@
 hello dojo, I'm Bruno, and my favourite codeschool path so far is git!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?